conmon-rs/server/src/rpc.rs

```diff
@@ -123,13 +124,18 @@ impl conmon::Server for Server {
             async move {
                 capnp_err!(container_log.write().await.init().await)?;
 
-                let grandchild_pid = capnp_err!(match child_reaper
+                let (grandchild_pid, token) = capnp_err!(match child_reaper
                     .create_child(runtime, args, &mut container_io, &pidfile)
                     .await
                 {
                     Err(e) => {
                         // Attach the stderr output to the error message
-                        let (_, stderr, _) = container_io.read_all_with_timeout(None).await;
+                        // TODO FIXME
```

Comment:
What need to be fixed here?

Reply:
ah I had to figure out if creating a new CancellationToken was problematic here. I think it's correct though.

Reply:
Closing the loop https://github.com/containers/conmon-rs/pull/699